src/System.Management.Automation/engine/MshMemberInfo.cs

```diff
@@ -2642,8 +2643,326 @@ public override string TypeNameOfValue
         /// True if the method is a special method like GET/SET property accessor methods.
         /// </summary>
         internal bool IsSpecial { get; private set; }
+
+
+        internal static PSMethod Create(string name, DotNetAdapter dotNetInstanceAdapter, object baseObject, DotNetAdapter.MethodCacheEntry method)
+        {
+            return Create(name, dotNetInstanceAdapter, baseObject, method, false, false);
+        }
+
+        internal static PSMethod Create(string name, DotNetAdapter dotNetInstanceAdapter, object baseObject, DotNetAdapter.MethodCacheEntry method, bool isSpecial, bool isHidden)
```

Comment:
I'm considering imposing a limit on line length :) - working on a laptop this week and GitHub isn't letting me scroll to see the rest of this line.

Reply:
4k 27" monitor may have desensitized me to the issue :)